server/bin/gold/test-11.txt

```diff
@@ -122,11 +114,6 @@ drwxrwxr-x          - public_html/users
 --- pbench tree state
 +++ pbench-local tree state (/var/tmp/pbench-test-server/test-11/pbench-local)
 drwxrwxr-x          - archive.backup
-drwxrwxr-x          - archive.backup/controller
--rw-r--r--    1558168 archive.backup/controller/fio__2016-08-16_22:03:11.tar.xz
--rw-r--r--         66 archive.backup/controller/fio__2016-08-16_22:03:11.tar.xz.md5
--rw-r--r--       7028 archive.backup/controller/pbench-user-benchmark_38_2016-05-18_19:36:32.tar.xz
--rw-r--r--         86 archive.backup/controller/pbench-user-benchmark_38_2016-05-18_19:36:32.tar.xz.md5
```

Comment:
The archive backup state was removed from the `test-11.tar.xz` tar ball because it just gets carried over and not part of this test.  The test itself runs `pbench-sync-satellite satellite-one` which does not do anything with backups.